assets/src/i18n/translateWithMarkup.js

```diff
@@ -121,6 +121,7 @@ function TranslateWithMarkup({ mapping = {}, children }) {
   return transform(
     new DOMParser().parseFromString(children, 'text/html').body.firstChild,
     mapping
+    //eslint-disable-next-line react/no-array-index-key
   ).map((element, index) => <Fragment key={index}>{element}</Fragment>);
```

Comment:
Could we just a `uuid` here?

Reply:
Not needed. The size and order of this array is not changing, so it's OK to use the array index in this case.